barber-shop/src/components/button/button.tsx

```diff
@@ -0,0 +1,15 @@
+import { HTMLInputTypeAttribute, InputHTMLAttributes, useState  } from 'react';
+
+import styles from './button.module.scss';
+
+interface Props extends InputHTMLAttributes<HTMLInputElement> {
+    isPassword?: boolean;
```

Comment:
acho que não vai ser necessária essas props, se for ter props tem que ser `active` e do tipo **boolean**, para quando o botão estiver ativo ou não